build.sbt

```diff
@@ -4,19 +4,17 @@ import sbt._
 import codacy.libs._
 import sbt._
 
-Universal / javaOptions ++= Seq("-Xms1g", "-Xmx2g", "-Xss512m", "-XX:+UseG1GC", "-XX:+UseStringDeduplication")
```

Comment:
are we sure about removing these?

Reply:
Yes, because these are inside the `root` project and now Sbt `1.6.0`gives a warning when you have unused settings. The real settings are in the `codacyAnalysisCli` project.
To make them global you need to add a `ThisBuild /` but I'm not going to do it since the cli has been working fine.